build.gradle.kts

```diff
@@ -110,6 +105,13 @@ dependencies {
 
 tasks.withType<JavaCompile> {
     options.encoding = "UTF-8"
+    options.release.set(8)
```

Comment:
So this replaces the source/target compatibility?

Reply:
Correct